src/GameSync.Api.Integration/GlobalSuppressions.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("StyleCop.CSharp.NamingRules", "SA1309:FieldNamesMustNotBeginWithUnderscore", Justification = "Reviewed.")]
+[assembly: SuppressMessage("StyleCop.CSharp.OrderingRules", "SA1206:DeclarationKeywordsMustFollowOrder", Justification = "Reviewed.")]
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1633:FileMustHaveHeader", Justification = "Reviewed.")]
+[assembly: SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1101:PrefixLocalCallsWithThis", Justification = "Reviewed.")]
```

Comment:
Please clear StyleCop's warnings. You can omit files which are examples/demos and won't be used in the final product.

Reply:
Almost all files are 'example' files, that would be overwritten or even deleted. I think it would be a waste of time to refactor this code at the moment. In those files we care only about logic (as the example of what goes where).